ShimmerAPI/ShimmerAPI/ShimmerBluetooth.cs

```diff
@@ -4828,7 +4828,7 @@ protected void SetCompatibilityCode()
                 }
                 else if (FirmwareIdentifier == ShimmerBluetooth.FW_IDENTIFIER_LOGANDSTREAM)   //LogAndStream
                 {
-                    if ((FirmwareMajor == 0 && FirmwareMinor >= 13 && FirmwareInternal >= 7))
+                    if ((FirmwareMajor == 0 && FirmwareMinor >= 14) || (FirmwareMajor == 0 && FirmwareMinor == 13 && FirmwareInternal >= 7)) //(FirmwareVersion >= 0.13.7)
```

Comment:
@MAzalya check isn't complete. If we release a LogAndStream v1.00.000 then this check will fail

Reply:
I suggest porting in the Java function and calling it